cdap-app-templates/cdap-etl/cdap-data-streams-base/src/main/java/io/cdap/cdap/datastreams/DataStreamsPipelineSpecGenerator.java

```diff
@@ -47,14 +54,21 @@
   private static final Gson GSON = new GsonBuilder()
                                      .registerTypeAdapter(Schema.class, new SchemaTypeAdapter())
                                      .create();
+  // Can be used as a runtime argument for the pipeline to disable at least once processing.
+  private static final String CDAP_STREAMING_ATLEASTONCE_ENABLED = "cdap.streaming.atleastonce.enabled";
```

Comment:
can you move this to the Constants class in cdap-etl-common?

Reply:
Done.